### PR TITLE
[YUNIKORN-590] GetPlaceholderTimeoutParam: undefined param is not an error

### PR DESCRIPTION
**Pull request by 0yukali0**
### What is this PR for?
It isn't an error that schedulingPolicyParameters is undefined.
It should return nil.

### What type of PR is it?
* [x] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-590

### How should this be tested?
I add TestGetPlaceholderTimeoutParam in gang_utils_test.go to observe its behavior.

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: pkg/common/utils/gang_utils_test.go [hunk 1]
```diff
@@ -289,3 +289,45 @@ func TestGetTaskGroupFromAnnotation(t *testing.T) {
 	assert.Equal(t, taskGroups2[0].MinResource["cpu"], resource.MustParse("2"))
 	assert.Equal(t, taskGroups2[0].MinResource["memory"], resource.MustParse("1Gi"))
 }
+
+func TestGetPlaceholderTimeoutParam(t *testing.T) {
```
Comment (yangwwei) [inline pkg/common/utils/gang_utils_test.go:293]: NIT: can we optimize this to be table-driven style, you can find some doc here: https://gobyexample.com/testing.

**Review by yangwwei — CHANGES_REQUESTED** [review]
Looks good, just add a NIT comment, pls check

File: pkg/common/utils/gang_utils_test.go (hunk 1 above)
Comment (0yukali0) [reply pkg/common/utils/gang_utils_test.go:293]: Understood.
I will optimize this with table-driven style.

**Review by yangwwei — APPROVED** [review]
looks good, waiting for test result.

**Comment by yangwwei** [comment]
All good, committing now, thanks @0yukali0

**Pull request merged**